src/python/tests/fasta/test_fasta_prep_data.py

```diff
@@ -22,120 +22,113 @@
 from contextlib import nullcontext as does_not_raise
 import filecmp
 from pathlib import Path
-from typing import ContextManager, List, Set
+from typing import ContextManager, Set
 
 import pytest
 
 import ensembl.io.genomio.fasta.process as FastaProcessing
 
 
-class TestFastaProcess:
```

Comment:
I thought I was following best practice to implement via a class. Any one explain why its better to remove said class ?

Reply:
Jorge Quote -> "So whilst the class approach is the best one when a setup is required for a set of tests, in your case you can use the two fixtures directly in your tests without the need of additional configuration. Thus why it can be removed. But I would suggest to always start with a class, and only remove it when you realise it is not required."